data/json/items/comestibles/drink.json

```
@@ -969,7 +969,7 @@
     "symbol": "~",
     "quench": 40,
     "calories": 2,
-    "description": "The beverage of gentlemen everywhere, made from applying hot water to leaves of the tea plant /Camellia sinensis/.",
+    "description": "The beverage of gentlemen everywhere, made from applying hot water to leaves of the tea plant /Camellia sinensis/.  Stronger than green tea.",
```

```suggestion
    "description": "The beverage of gentlemen everywhere, made from applying hot water to oxidized leaves of the tea plant /Camellia sinensis/.",
```

I don't think the change here adds anything